web-ui/src/main/resources/catalog/style/gn_editor.less

```diff
@@ -4,7 +4,8 @@
 @import (less) "../lib/style/datetimepicker.css";
 @import (less) "../lib/jquery.ext/jquery.fileupload-ui.css";
 @import "gn.less";
-
+@color_1: grey;
+@color_2: green;
```

Comment:
would be better to use bootstrap color variables ? 
eg. gray and brand-primary ? https://github.com/twbs/bootstrap/blob/a365d8689c3f3cee7f1acf86b61270ecca8e106d/less/variables.less#L12

if not more meaningful names would be better.